src/main/java/core/basesyntax/BallCollors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallCollors {

```

Comment:
enum name should be singular and don't specify ball here
```suggestion
public enum Color {

```